doc/api/index.rst

```diff
@@ -138,6 +138,9 @@ Operations on grids
     grdfill
     grdfilter
     grdgradient
+    grdhisteq
+    grdhisteq.equalize_grid
+    grdhisteq.compute_bins
```

Comment:
I'll modify this to keep the top-level class.

Reply:
I actually couldn't get autodoc to generate the pages for equalize_grid or compute_bins without including them in the toctree. I might look more into it later, but I also don't think it would be that big of deal to list all the class methods.